lib/inspec/config.rb

```diff
@@ -67,6 +67,10 @@ def diagnose
       puts
     end
 
+    def telemetry_enabled?
+      final_options['telemetry']
+    end
+
```

Comment:
I felt it was better to add an accessor here rather than our collector needing to know the format of our options hash and reaching inside.

Reply:
I'm unsure about this. Should the config object have *general* knowledge of the  major subsystems of InSpec ? I think yes (it already know about train credentials, for example). 

However, asking specifically if the telemetry system is enabled means that when you add another config option - like endpoint URL - you'll need to add another method to the Config (or else scatter such methods between the Collector and Config).

So, I think if you asked the Config, "I don't care how you got it, but can you give me all the Telemetry options as a hash" would be fine - Collector doesn't have to care about Config's innards, and vice-versa.

Reply:
I'm unsure about this. Should the config object have *general* knowledge of the  major subsystems of InSpec ? I think yes (it already know about train credentials, for example). 

However, asking specifically if the telemetry system is enabled means that when you add another config option - like endpoint URL - you'll need to add another method to the Config (or else scatter such methods between the Collector and Config).

So, I think if you asked the Config, "I don't care how you got it, but can you give me all the Telemetry options as a hash" would be fine - Collector doesn't have to care about Config's innards, and vice-versa.